src/backend/mod.rs

```diff
@@ -66,7 +66,7 @@ pub enum Event {
     GlobalRemoved(u32),
     GlobalInfo(u32, Box<[(&'static str, String)]>),
     GlobalProperties(u32, std::collections::BTreeMap<String, String>),
-    ClientPermissions(u32, u32, Vec<pw::permissions::Permission>),
+    ClientPermissions(u32, Vec<pw::permissions::Permission>),
```

Comment:
```suggestion
    #[allow(dead_code)]
    ClientPermissions(u32, u32, Vec<pw::permissions::Permission>),
```
I'd like to keep the variants of this enum as similar to PipeWire's messages as possible

Reply:
I put the allowance for dead code only relative to the index (the second field).